cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleService.java

```diff
@@ -1000,6 +1005,7 @@ private ApplicationWithPrograms deployApp(NamespaceId namespaceId, @Nullable Str
     ApplicationWithPrograms applicationWithPrograms;
     try {
       applicationWithPrograms = manager.deploy(deploymentInfo).get();
+      emitMetricForMultipleSourcePlugins(isPreview, applicationWithPrograms);
```

Comment:
we don't want to emit this metric here, as CDAP doesn't know what a "streaming source" is. The DataStreamsApp is the one that knows what a streaming source is and has to be the one emitting metrics. For example, in the SparkStreamingPipelineRunner we can emit a metric. The thing to note though is that it would be a metric that is emitted every run, instead of number of pipeline with multiple sources.